frontend/src/components/react-table/table.tsx

```diff
@@ -0,0 +1,66 @@
+import { ReactElement } from 'react';
+import { ReactTableProps } from 'common/types/react-table/react-table-props';
+import { useTable } from '../../hooks/use-table/use-table';
+import { ReactTableRenderEntitiesNames } from 'components/common/react-table/react-table-render-entities-names';
+import { ClassNames } from './common/class-names';
+
+const Table = ({
+  tableId,
+  columnsData,
+  rowsData,
+}: ReactTableProps): ReactElement => {
```

Comment:
```suggestion

type Props = {
  columns: Column[]; // from react-table package
  data: unknown[];
}

const Table: FC<Props> = ({
  columns,
  data,
}) => {
```
remove all useless types